app/decorators/identity_decorator.rb

```diff
@@ -11,6 +11,10 @@ def return_to_sp_url
   end
 
   def happened_at
-    EasternTimePresenter.new(identity.last_authenticated_at).to_s
+    identity.last_authenticated_at
+  end
+
+  def happened_at_in_words
+    EasternTimePresenter.new(happened_at).to_s
```

Comment:
Do we want to inherit these two decorators (user, identity) from a common ancestor and place the shared `happened_at_in_words` method there?